crates/core/src/db/update.rs

```diff
@@ -1,627 +1,220 @@
 use super::datastore::locking_tx_datastore::MutTxId;
 use super::relational_db::RelationalDB;
 use crate::database_logger::SystemLogger;
-use crate::error::{DBError, TableError};
-use anyhow::Context;
-use core::{fmt, mem};
-use enum_as_inner::EnumAsInner;
-use itertools::Itertools;
-use similar::{Algorithm, TextDiff};
+use crate::execution_context::ExecutionContext;
```

Comment:
This file was completely rewritten so should probably not be reviewed in diff view.